core/feature_flag_list.py

```diff
@@ -99,7 +97,6 @@ class FeatureNames(enum.Enum):
     FeatureNames.DUMMY_FEATURE_FLAG_FOR_E2E_TESTS,
     FeatureNames.END_CHAPTER_CELEBRATION,
     FeatureNames.CHECKPOINT_CELEBRATION,
-    FeatureNames.CONTRIBUTOR_DASHBOARD_ACCOMPLISHMENTS,
```

Comment:
You need to move this to the deprecated features list below, not just delete it

Reply:
Thanks for the review, I'll look through this at my earliest

Reply:
hi, just added it to the list & checked through all possible cases